cmd/clusterctl/api/v1alpha3/annotations.go

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright 2019 The Kubernetes Authors.
```

Comment:
Nit, but with a new file, should probably make sure to use a current copyright year.